### PR TITLE
fix start directory

### LINKED ISSUE #9: Session StartDirectory is ignored
```go
package main

import (
	"fmt"

	gotmux "github.com/jubnzv/go-tmux"
)

func main() {
	// Create instance of the running tmux server.
	server := new(gotmux.Server)

	// Prepare a configuration for the new session that contains some windows.
	session := gotmux.Session{Name: "windows-panes", StartDirectory: "/tmp/"}
	w := gotmux.Window{Name: "default-tmp", Id: 0}

	session.AddWindow(w)
	server.AddSession(session)
	sessions := []*gotmux.Session{}
	sessions = append(sessions, &session)
	conf := gotmux.Configuration{
		Server:        server,
		Sessions:      sessions,
		ActiveSession: nil,
	}

	// Setup this configuration.
	err := conf.Apply()
	if err != nil {
		msg := fmt.Errorf("Can't apply prepared configuration: %s", err)
		fmt.Println(msg)
		return
	}

	currentWindows, _ := session.ListWindows()
	for _, w := range currentWindows {
		fmt.Printf("Current window: %v\n", w)
		currentPanes, _ := w.ListPanes()
		fmt.Printf("Current panes: %v\n", currentPanes)
		fmt.Println()
	}

	// Attach to the created session
	err = session.AttachSession()
	if err != nil {
		msg := fmt.Errorf("Can't attached to created session: %s", err)
		fmt.Println(msg)
		return
	}
}

```

```
Current window: {default-tmp 1 0 windows-panes /madflow/go-tmux []}
```

I would expect the windows directory to be "/tmp/".

### PR DESCRIPTION
**Pull request by madflow**
* Fixes #9

### CODE REVIEW

File: configuration.go [hunk 1]
```diff
@@ -84,18 +86,29 @@ func (c *Configuration) Apply() error {
 
 		// Add windows for created session
 		for _, w := range s.Windows {
+
```
Comment (jubnzv) [inline configuration.go]: Please remove empty newlines

File: configuration.go [hunk 2]
```diff
@@ -72,8 +72,10 @@ func (c *Configuration) Apply() error {
 			attached_key,
 			"-n", initial_window.Name,
 			"-D", // If session with same name exists, attach to it
-			"-s", s.Name}
+			"-s", s.Name,
+		}
 		args = append(args, args_start_dir...)
+
```
Comment (jubnzv) [inline configuration.go]: And here

**Comment by jubnzv** [comment]
Thanks! It seems valid for me. Please fix the codestyle, and we will merge it.

**Comment by jubnzv** [comment]
Thank you! It seems valid for me.
We can merge it after fixing the codestyle.

**Comment by madflow** [comment]
Rebased

**Comment by jubnzv** [comment]
Thank you!

**Pull request merged**